src/pages/resultsView/pathwayMapper/ResultsViewPathwayMapper.tsx

```diff
@@ -201,6 +207,46 @@ export default class ResultsViewPathwayMapper extends React.Component<
                                 store={this.props.store}
                                 tabReflectsOql={true}
                             />
+                            <div
```

Comment:
this is a lot of logic here so either make this a functional component or put it in a @computed getter of the component.